.changeset/clever-doors-compare.md

```diff
@@ -0,0 +1,77 @@
+---
+'@razorpay/blade': major
+---
+
+feat(Dropdown): Controlled Dropdown and Button Trigger
```

Comment:
A brief explanation of the major release, eg:

```suggestion
feat(Dropdown): Controlled Dropdown and Button Trigger
- Adds APIs to seamlessly build controlled dropdown
- Adds ways to use dropdown as a menu - using a dropdown button as a trigger
- Removes `isDefaultSelected` from `ActionListItem` *(see migration guide)*
```

Reply:
done